packages/backend/src/services/DashboardService/DashboardService.ts

```diff
@@ -406,12 +412,17 @@ export class DashboardService extends BaseService {
                 }),
             );
 
+            const newTabbedTiles = updatedTiles.map((tile) => ({
+                ...tile,
+                tabUuid: newTabs.find((tab) => tab.uuid === tile.tabUuid)?.uuid,
```

Comment:
Isn't this always `undefined` ?  the `tile.tabUuid` has uuids from the original dashboard and the `newTabs` has the newly generated uuids. So there is never a match 🤔 

We need a `tabUuidMap` that tracks the old and new uuids. 
We can also move this inside the `updatedTiles` loop.